### ISSUE TITLE
Plex Ratelimit (?)

### ISSUE THREAD

**Issue by BLACK4585** [Taxel/PlexTraktSync#2106]
### Confirmation

- [X] I have read the [README.md](https://github.com/Taxel/PlexTraktSync#readme) on the project homepage
- [X] I have checked if identical issue already exists
- [X] I have tried downgrading to find version that can be used as a workaround

### The problem

https://github.com/Taxel/PlexTraktSync/discussions/2104
Just linking my already written infos, if you need something extra please tell me.

### Steps to reproduce the behavior

https://github.com/Taxel/PlexTraktSync/discussions/2104

### Expected behavior

To sync properly.

### Install method

pipx

### Version

0.32.2

### Python Version

Python Version: 3.12.7 (tags/v3.12.7:0b05ead, Oct  1 2024, 03:06:41) [MSC v.1941 64 bit (AMD64)]

### Plex Server Version

Plex Server version: 1.41.3.9232-434341a9d, updated at: 2024-12-03 09:41:46

### Operating System and Version

Windows 11

**Comment by buugaas** [comment]
getting the same issues suddenly. switched to latest nightly and same
plex trakt sync Rate Limit Exceeded for plextraktsync.trakt.TraktApi.search_by_id

**Comment by buugaas** [comment]
Yes exactly the same here.  Issue started about 3 days ago for me.

**Comment by devnull34** [comment]
https://forums.trakt.tv/t/has-the-trakt-api-rate-limit-changed/40054

**Comment by carlosmg2** [comment]
+1, same issue. plex trakt sync Rate Limit Exceeded for plextraktsync.trakt.TraktApi

**Comment by glensc** [comment]
Closing and locking. One says Plex rate limit, the other says Trakt rate limit and thinking it's "the same". It is not.

If you want proper feedback, report your errors, not let others follow trees of links.
